playbooks/roles/figures/tasks/main.yml

```
@@ -56,15 +56,25 @@
     - figures
     - deploy:figures
 
-- name: install figures from {{ figures_code_dir }}
+- name: install figures {{ figures_version }} from {{ figures_code_dir }}
   args:
     chdir: "{{ figures_code_dir }}"
   command: "{{ edxapp_venv_dir }}/bin/pip install -e ."
   become_user: "{{ edxapp_user }}"
   environment: "{{ edxapp_environment }}"
   tags:
     - figures
-    - install:figures
+    - figures:install
+
+- name: migrate figures
+  command: "{{ edxapp_venv_bin }}/python manage.py lms migrate figures --settings={{ EDXAPP_SETTINGS }}"
```

See https://github.com/edx/configuration/pull/4812/files for more info on why this is needed.

```suggestion
  command: ". {{ edxapp_app_dir }}/edxapp_env && python manage.py lms migrate figures --settings={{ EDXAPP_SETTINGS }}"
```

---

@OmarIthawi thanks for pointing this out. I've pushed the changes.